device/iot-device-client/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/device/MessagePropertyTest.java

```diff
@@ -46,32 +46,6 @@ public void constructorSavesPropertyValue()
 		
     }
 
-	 @Test
-    public void constructorSavesPropertyValueSlash()
```

Comment:
These tests were incorrect, as '/' characters are not allowed in application properties